packages/p2p-v2/src/components/BuySellForm/BuySellFormFooter/BuySellFormFooter.tsx

```diff
@@ -4,16 +4,22 @@ import { Button, useDevice } from '@deriv-com/ui';
 type TBuySellFormFooterProps = {
     isDisabled: boolean;
     onClickCancel: () => void;
-    onSubmit: () => void;
+    onSubmit?: () => void;
 };
-const BuySellFormFooter = ({ isDisabled, onClickCancel, onSubmit }: TBuySellFormFooterProps) => {
+const BuySellFormFooter = ({
+    isDisabled,
+    onClickCancel,
+    onSubmit = () => {
+        //do nothing
+    },
+}: TBuySellFormFooterProps) => {
     const { isMobile } = useDevice();
     return (
-        <div className='flex justify-end gap-[1rem] p-[1.6rem]'>
+        <div className='flex justify-end gap-[1rem]'>
             <Button onClick={onClickCancel} size='lg' textSize={isMobile ? 'md' : 'sm'} variant='outlined'>
                 Cancel
             </Button>
-            <Button disabled={isDisabled} onClick={onSubmit} size='lg' textSize={isMobile ? 'md' : 'sm'}>
+            <Button disabled={isDisabled} onClick={onSubmit} size='lg' textSize={isMobile ? 'md' : 'sm'} type='submit'>
```

Comment:
Does this work @nada-deriv? instead of the `"// do nothing"` above? 
```suggestion
            <Button disabled={isDisabled} onClick={(e) => onSubmit?.(e)} size='lg' textSize={isMobile ? 'md' : 'sm'} type='submit'>
```